### ISSUE TITLE
Explain what the different actions do

### ISSUE THREAD

**Issue by jancborchardt** [CyborgUnplug/CyborgUnplug#30]
In the screen where it says »Quick sweep« etc. all the options I couldn’t really understand from the name what they would result in, or what scenario they are ideal for. Some kind of explanation would be quite good.

**Comment by jancborchardt** [comment]
Right, the step I’m referring to is »Detect spy devices: Screen 2«:
![](https://plugunplug.net/guide/img/14.jpg)
In the guide there’s good explanations, but it’s quite cumbersome to always have to have the guide open next to it, or need to remember.

**Comment by bnvk** [comment]
Thanks @jancborchardt this is good feedback!
